doc/source/reference/arrays.datetime.rst

```diff
@@ -360,37 +363,3 @@ Some examples::
     # any amount of whitespace is allowed; abbreviations are case-sensitive.
     weekmask = "MonTue Wed  Thu\tFri"
 
-Changes with NumPy 1.11
-=======================
```

Comment:
Does it make sense to move this to the 1.11 release notes?

Reply:
It is already in the release notes.

Reply:
For 1.11 I mean.

Reply:
You're right, it's already  verbatim[here](https://github.com/numpy/numpy/blob/master/doc/source/release/1.11.0-notes.rst#datetime64-changes).